app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -13,8 +13,13 @@ def index
 
       def addresses(search_term)
         suggestion = geocoder.suggest(search_term).first
+        # this happens multiple times
+        if !suggestion
```

Comment:
if we input an address like `PO Box 168, Yellowstone National Park, WY 82190-0168` arcgis will not find an appropriate suggestion. We want to log this in analytics. That happens successfully but multiple times rather than a single time.